### PR TITLE
BULK ORDERING - Edit Catalogue Solution - Flat Variable (On demand) -…

### PR DESCRIPTION
**Pull request by kiran-mastek**
… Post save Catalogue Solution

Updated template to add Order Item Id

Story [AB#9546](https://buyingcatalog.visualstudio.com/c5f97979-5b03-4d10-ba8d-871d0526b408/_workitems/edit/9546)
Task [AB#12133](https://buyingcatalog.visualstudio.com/c5f97979-5b03-4d10-ba8d-871d0526b408/_workitems/edit/12133)

### CODE REVIEW

**Comment by bjss-andrew** [comment]
this should prob be merged into development not master if its bulk work

File: app/components/general/components/bc-table/macro.njk [hunk 1]
```diff
@@ -73,7 +73,8 @@
               {{ viewDataText({
                 dataTestId: dataPoint.dataTestId,
                 classes: dataPoint.classes,
-                data: dataPoint.data
+                data: dataPoint.data,
+                dataOrderItemId: dataPoint.dataOrderItemId if dataPoint.dataOrderItemId
```
Comment (bjss-andrew) [inline app/components/general/components/bc-table/macro.njk:77]: is the `if` needed? wont it be undefined either-way if it dataPoint doesnt have a dataOrderItemId property?

**Comment by bjss-chris** [comment]
> this should prob be merged into development not master if its bulk work

There isn't a development branch of the component library. However, I think we should hang fire on this one as it relates to the bulk order items bug that I'm working on.

File: app/components/general/components/bc-table/macro.njk (hunk 1 above)
Comment (kiran-mastek) [reply app/components/general/components/bc-table/macro.njk:77]: Without the if statement the template fails to render when the property is not present, on list orders page

**Review by bjss-andrew — APPROVED** [review]

**Pull request merged**